### PR TITLE
Epoundor/dragndrop

### PR DESCRIPTION
**Pull request by epoundor**

### CODE REVIEW

**Review by YacineSteeve — APPROVED** [review]
There are some little details I'm not very cumfortable with, but let's go with it for now 🙂
Good job

File: apps/frontend/src/components/Decorator/Draggable.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from 'react';
```
[the remark this answers is not in the document]
Comment (epoundor) [reply apps/frontend/src/components/Decorator/Draggable.tsx:1]: I don't knew it.
How do you do it?

File: apps/frontend/index.html [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-    <div id="root"></div>
-    <script type="module" src="./src/main.tsx"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://cdn.tailwindcss.com"></script>
```
[the remark this answers is not in the document]
Comment (epoundor) [reply apps/frontend/index.html:8]: Exactly

File: apps/frontend/src/components/Decorator/Draggable.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (YacineSteeve) [reply apps/frontend/src/components/Decorator/Draggable.tsx:1]: Misstelling, it is automatically done by `vite/plugin-react`

**Pull request merged**